### ISSUE TITLE
Shaking Text on Install Tab

### ISSUE THREAD

**Issue by xyhbhtt** [ChrisTitusTech/winutil#1390]
**Describe the bug**
The Text in the Install Tab is shaking and seems like a render mistake

**To Reproduce**
Open Winutil

**Expected behavior**
No jittering text

**Screenshots**
Great examples are Ferdium, Guilded and Hexchat
![jitter](https://github.com/ChrisTitusTech/winutil/assets/142528095/90f2a343-91d2-4a3d-a8f6-6283df6abf22)

**Comment by KonTy** [comment]
please try the new version

```
iwr -useb https://raw.githubusercontent.com/ChrisTitusTech/winutil/main/winutil.ps1 | iex
``

**Comment by xyhbhtt** [comment]
The Communications segment is shaking much less, or just with a way lower frequency, while the res is now shaking in unisence.
![jit2](https://github.com/ChrisTitusTech/winutil/assets/142528095/6359127f-3b38-4395-8ef6-3c13374371fd)

**Comment by KonTy** [comment]
Are you saying the text is shaking? The text is not shaking in that picture, I dont see that. I see the shadow on the box slowly glowing. that's by design, it is a glowing effect

**Comment by xyhbhtt** [comment]
I say that the text is a shifting back and forth. That doesn't look like a glowing effect to me.
![jit3](https://github.com/ChrisTitusTech/winutil/assets/142528095/06595284-38bf-4a8a-80a4-742583f40ebd)

**Comment by KonTy** [comment]
this is so strange I'm not seeing it :) do you see this with the white theme?

**Comment by xyhbhtt** [comment]
Nope. The light theme is fine. And before you might ask, I've tried it on a second computer and had same display characteristics. just a bit slower.

**Comment by Mavromatis** [comment]
I can second that visual. I wondered if it's my eyes but seeing it here I'm now sure that it shakes a bit on mouseover. Maybe due to change in redrawing that line, green to white, have some redrawing effect on the page.
